tests/helper.ts

```
@@ -128,25 +134,22 @@ export function getApis(apisNames: string[], fileName: string): Api[] {
 
-export function getJSON(
-  fileName: string,
-  { isRequest }: { isRequest: boolean }
-): {} {
+export function getJSON({
+  fileName,
+  fileFolder,
+}: {
+  fileName: string;
+  fileFolder: "request-bodies" | "response-bodies" | "lookup-data";
+}): Record<string, {}> | string[] {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)